pkg/volume/local/local.go

```diff
@@ -207,6 +213,17 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	if !notMnt {
 		return nil
 	}
+	if fsGroup != nil {
+		fsGroupSame, fsGroupOld, err := isSamefsGroup(m.globalPath, fsGroup)
```

Comment:
So this is going to require that the local volume is already initially created by the admin with the correct fsGroup.  Is that the requirement we want to impose?